source/_integrations/derivative.markdown

```diff
@@ -12,16 +12,35 @@ ha_qa_scale: internal
 ha_codeowners:
   - '@afaucogney'
 ha_domain: derivative
+ha_config_flow: true
 ha_platforms:
   - sensor
 ---
 
-The `derivative` platform creates a sensor that estimates the derivative of the values provided by a source sensor.
-Derivative sensors are updated upon changes of the **source**.
+The derivative integration creates a sensor that estimates the derivative of the
+values provided by another source sensor. Derivative sensors are updated upon changes of the **source sensor** used.
 
-## Configuration
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Name:
+  description: The name the sensor should have. You can change it again later.
+Input sensor:
```

Comment:
Should we call it the "source sensor" instead?

Reply:
🤷  I think of it as an input (which matched terminology we use with Blueprints I guess). But, either way, is fine IMHO. Currently, it matches the Core PR.